src/styles/blocks/calendar.scss

```
@@ -0,0 +1,40 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-box-gap;
+  max-width: $conrainer-size;
+  padding: $container-padding;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: $box-background-size;
```

```suggestion
    background-color: $box-background-color;
```
and don't forget to rename in variables